cloud/amazon/ec2_snapshot.py

```diff
@@ -115,7 +128,128 @@
     HAS_BOTO = False
 
 
-def main():
+# Find the most recent snapshot
+def _get_snapshot_starttime(snap):
```

Comment:
I'm not convinced about prefixing methods with underscores in modules. I know it means they're private methods, but nothing really imports modules anyway.